### ISSUE TITLE
BN should be used after ReLU

### ISSUE THREAD

**Issue by yunlingz** [yunjey/pytorch-tutorial#212]
<https://github.com/yunjey/pytorch-tutorial/blob/master/tutorials/02-intermediate/convolutional_neural_network/main.py#L41-L42>

```python
nn.ReLU(),
nn.BatchNorm2d(...),
```

BN should be used after ReLU. Features might be truncated by non-linearity like ReLU, so BN is used to normalize the distribution of features.

**Comment by scalalang2** [comment]
I have been also suffered from this issue.

When ReLU is used after BN, Loss is not converged at all.

```
nn.BatchNorm2d(16),
nn.ReLU(),

...
Epoch [0/10], Step: [200/600], Loss: [0.39470914006233215]
Epoch [0/10], Step: [400/600], Loss: [0.23821884393692017]
Epoch [0/10], Step: [600/600], Loss: [0.1606658548116684]
...
Epoch [9/10], Step: [200/600], Loss: [0.010754228569567204]
Epoch [9/10], Step: [400/600], Loss: [0.047237977385520935]
Epoch [9/10], Step: [600/600], Loss: [0.06365286558866501]
Test Accuracy: [9.78]
```
